packages/usa-select/src/test/test-patterns/usa-select--multiple.twig

```diff
@@ -0,0 +1,13 @@
+<form class="usa-form">
+  <label class="usa-label" for="options">Dropdown label</label>
+  <select class="usa-select" name="options" id="options" multiple  size={{ size }} >
+    <option value>- Select -</option>
+    <option value="value1">Option A</option>
+    <option value="value2">Option B</option>
+    <option value="value3">Option C</option>
+    <option value="value1">Option E</option>
+    <option value="value2">Option F</option>
+    <option value="value3">Option G</option>
+  </select>
+</form>
+
```

Comment:
Can you double-check the empty lines at end of file? Locally shows 2 empty lines, should be 1.